report/3.Theory/LanguageSpecification.tex

```diff
@@ -76,6 +85,27 @@ \section{Language Specification}
 ["john", "frank"] where {value is "frank"} print
 \end{lstlisting}
 
+\textbf{Grammar for function calls}\\
+There are three different ways to call a function depending on how many parameters must be passed with the function:
+\begin{itemize}
+	\item If no parameters must be passed, then invoking the function identifier is a call in and of itself.
```

Comment:
"If the function has no parameters, then invokation of the function is done with the identifier alone. an example could be "printHelloWorld"